### PR TITLE
Adding docs in Vietnamese

### PR DESCRIPTION
**Pull request by truongnh1992**
Adding `kubectl` cheatsheet in Vietnamese

Signed-off-by: Nguyen Hai Truong <truongnh@vn.fujitsu.com>

### CODE REVIEW

File: content/vi/docs/reference/kubectl/cheatsheet.md [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+  username: $(echo -n "jane" | base64 -w0)
+EOF
+
+```
+
+## Xem, tìm các tài nguyên
+
+```bash
+# Lệnh get với một số đầu ra cơ bản
+kubectl get services                          # Liệt kê tất cả các dịch vụ trong namespace
```
Comment (ngtuna) [inline content/vi/docs/reference/kubectl/cheatsheet.md]: we shouldn't translate primitives to Vietnamese. Just call it `services`

File: content/vi/docs/reference/kubectl/cheatsheet.md [hunk 2]
```diff
@@ -0,0 +1,380 @@
...
+kubectl get pods --all-namespaces             # Liệt kê tất cả các pods trong tất cả các namespaces
+kubectl get pods -o wide                      # Liệt kê tất cả các pods namespace, với nhiều thông tin hơn
+kubectl get deployment my-dep                 # Liệt kê một triển khai cụ thể
```
Comment (ngtuna) [inline content/vi/docs/reference/kubectl/cheatsheet.md]: deployment

File: content/vi/docs/reference/kubectl/cheatsheet.md (hunk 1 above)
Comment (truongnh1992) [reply content/vi/docs/reference/kubectl/cheatsheet.md]: I've just updated the document to resolve your change request.

**Pull request merged**